layouts/index.html

```diff
@@ -1,10 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Edmonton Go: {{ .Title }}</title>
+  <link rel="canonical" href="{{ .Permalink }}">
+  <meta name="google-site-verification" content="EesR8siX5ZhCkifF0oZAk04ZDF-TXub5WDl-cAxyMVg" />
+  <link rel="stylesheet" href="/assets/all.css">
+  <link rel="icon" type="image/png" href="/favicon.png">
+</head>
+<body>
+<div class="site">
```

Comment:
Why duplicate this code instead of leaving it in the reused header.html? It looks identical.

Also, workshop/single.html no longer works. http://127.0.0.1:1313/workshop/

Reply:
i missed that one. I have followed your suggestions and added a partial called head which contains the html head and included that into the other templates. The workshop works again.